plugins/Denizen/scripts/climbing_chalk.dsc

```diff
@@ -0,0 +1,79 @@
+climbing_chalk:
+    type: item
+    debug: false
+    material: sugar
+    allow in material recipes: false
+    display name: <dark_aqua>Climbing Chalk
+    lore:
+    - <&7>Strengthens your grip.
+    - <&7>Allows you to climb on fences, inner walls, buttons, chains, and iron bars.
+    - <&7>Lasts 5 minutes.
+    - <&7>Sneak + Right Click in air to apply.
+    recipes:
+        1:
+            type: shapeless
+            output_quantity: 4
+            input: calcite
+
+climbing_chalk_handler:
```

Comment:
(All throughout this script) small nitpick, but maybe use a flag map of `climbing_chalk` and then have keys like `.active` etc